### PR TITLE
Add panorama9.com

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Panorama9 is a cloud-based IT asset management and monitoring platform that helps businesses manage and secure their devices, software, and networks. The platform provides real-time visibility, compliance management, and automated IT operations to streamline IT workflows. The platform offers 2FA for the account login verification process. The website is ranked 3,349,268 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=panorama9.com). The website provides documentation on how to turn on 2FA.
ps: Although it is not ranked high on SimilarWeb, it ranks 11,997 on Tranco.

image

### CODE REVIEW

File: entries/p/panorama9.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Panorama9": {
+    "domain": "panorama9.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "http://support.panorama9.com/articles/1859601",
+    "categories": [
+      "cloud"
```
Comment (Carlgo11) [inline entries/p/panorama9.com.json:9]: Remote Access might be a more fitting category here :thinking:
What do you think @Yang-Nankai?
```suggestion
      "remote"
```

File: img/p/panorama9.com.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 225 225"><path fill="#429BFF" d="M116.167 150.2c-18.708 14.202-37.865 14.248-58.08 2.504 0 11.944.007 23.205-.007 34.465-.002 1.83.113 3.697-.192 5.483-.823 4.813-5.216 8.25-9.988 8.028-5.036-.234-8.687-3.71-9.195-8.812-.115-1.157-.078-2.33-.078-3.496-.003-24.823-.039-49.646.009-74.469.041-21.458 12.806-39.452 33.31-45.266 20.51-5.816 41.416-.127 54.41 20.656 11.413 18.254 8.418 42.236-6.675 57.56-1.052 1.067-2.16 2.078-3.514 3.347M75.79 88.035c-1.489.74-3.027 1.394-4.458 2.232-10.73 6.283-15.724 19.916-11.794 32.088 3.73 11.551 15.963 20.293 27.45 19.208 11.752-1.11 20.448-6.977 24.7-18.162 4.029-10.6 1.927-20.496-5.912-28.616-8.094-8.384-18.01-10.865-29.986-6.75z"/><path fill="#439BFF" d="M170.902 29.95c15.768 9.734 20.777 25.395 13.467 41.963-6.779 15.366-13.764 30.64-20.726 45.924-3.062 6.723-8.36 9.092-13.945 6.433-5.53-2.633-6.92-8.08-3.847-14.907 2.91-6.466 5.782-12.949 8.707-19.506-24.185-6.564-34.984-23.473-27.924-42.864 6.501-17.859 26.126-25.538 44.268-17.043m-19.17 41.077c8.518 2.029 15.997-2 17.99-9.69 1.877-7.239-2.293-14.71-9.452-16.937-6.856-2.132-14.3 1.489-17.129 8.33-2.86 6.92.218 14.007 8.591 18.297z"/></svg>
```
Comment (Carlgo11) [inline img/p/panorama9.com.svg]: Same image but without the padding.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 149.1 174.2"><path fill="#429bff" d="M132.3 3.5c15.8 9.7 20.8 25.4 13.5 42C139 60.7 132 76 125 91.3c-3 6.7-8.3 9-14 6.4-5.4-2.6-6.8-8.1-3.8-15 3-6.4 5.8-12.9 8.7-19.4C91.8 56.8 81 39.9 88 20.5c6.5-17.9 26.1-25.5 44.3-17m-19.2 41c8.5 2 16-2 18-9.6a14 14 0 0 0-9.4-17 14.1 14.1 0 0 0-17.2 8.3c-2.8 7 .2 14 8.6 18.3Zm-35.5 79.2c-18.8 14.2-38 14.3-58.1 2.5v34.5c0 1.8 0 3.7-.2 5.5a9.8 9.8 0 0 1-10 8c-5-.2-8.7-3.7-9.2-8.8v-78a45.7 45.7 0 0 1 33.2-45.2 46.1 46.1 0 0 1 54.4 20.6 47 47 0 0 1-6.6 57.6l-3.5 3.3M37.2 61.5c-1.5.8-3 1.4-4.5 2.3a27.6 27.6 0 0 0-11.8 32c3.8 11.6 16 20.4 27.5 19.3C60 114 68.8 108 73 96.9c4-10.6 1.9-20.5-6-28.6-8-8.4-18-10.9-30-6.8"/></svg>
```

Comment (Yang-Nankai) [reply img/p/panorama9.com.svg]: Thanks!

File: entries/p/panorama9.com.json (hunk 1 above)
Comment (Yang-Nankai) [reply entries/p/panorama9.com.json:9]: I think based on the function, this website should belong to remote access. If based on the website's method, I think it belongs to the cloud category. 😕 @Carlgo11 

image

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**